src/collections/_documentation/workflow/alerts-notifications/alerts.md

```diff
@@ -97,14 +97,17 @@ In the example, the alert will not fire because an event cannot satisfy both of
 
 The following actions are available:
 
+- Send an email
+    - Send email to either [Issue Owners]({%- link _documentation/workflow/issue-owners.md -%}), Team, or [Member]({%- link _documentation/accounts/membership.md -%}#member).
 - Send a notification to all legacy integrations
-    - [Legacy integrations]({%- link _documentation/workflow/integrations/legacy-integrations.md -%}), also known as Plugins, are configured per project. Legacy integrations currently include email.
-- Send a notification to a single legacy integration (service)
+    - [Legacy integrations]({%- link _documentation/workflow/integrations/legacy-integrations.md -%}), also known as Plugins, are configured per project. 
+- Send a notification to one legacy integration (service)
     - "service" is the same as legacy integration
+    - If no legacy integrations are enabled, this option is hidden.
```

Comment:
```suggestion
    - If no legacy integrations or integrations built using the integration platform are enabled, this option is hidden.
```
You can wordsmith it but that's the general gist.

The unfortunately part of this grouping is that this also includes published apps. And our users don't know which published apps are built on our platform vs built by us. More of a heads up for @adhiraj.